packages/fx-core/src/common/azure-hosting/botServiceHosting.ts

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AzureHosting } from "./azureHosting";
 import { ServiceType } from "./interfaces";
+import { AzureService } from "./azureService";
 
-export class BotServiceHosting extends AzureHosting {
+export class BotServiceHosting extends AzureService {
```

Comment:
Same here, the class name could be "AzureBotService" as well as the filename, according to Azure service official documents https://azure.microsoft.com/en-us/services/